src/app/(marketing)/pricing/components/PricingCards.module.css

```
@@ -0,0 +1,414 @@
+    opacity: 0;
+    transform: translate3d(0, 1.75rem, 0) rotateX(0deg) rotateY(0deg);
+  }
+  to {
+    opacity: 1;
+    transform: translate3d(0, var(--card-lift), 0) rotateX(var(--card-tilt-x))
+      rotateY(var(--card-tilt-y));
+  }
+}
+
+/* Opaque card face — sits above the under-glow. */
+.cards :global(.cl-pricingTableCard)::after {
```

Stylelint (17.14.1)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)